src/demo/models/guide.js

```diff
@@ -95,13 +92,13 @@ const guides = [
     }
   },
   {
-    id: 'fishvtrash-predicting-init',
-    text: `Now let's see if A.I. knows what a "fish" looks like.`,
+    id: 'fishvtrash-predicting-init1',
+    text: `Now let's see if A.I. knows what a "Fish" looks like.`,
```

Comment:
nit: We've used different double quotes above.. we should be consistent, though it doesn't matter too much which we choose.

Reply:
I just pushed an update to address this (i believe) so that anywhere we show the word that AI was being trained on we surround it with " ". Let me know if that addresses your concern. I also fixed one location where I was using the wrong outer quotes. Thanks.